protector/proxy/request_handler.py

```diff
@@ -31,6 +32,12 @@ class ProxyRequestHandler(BaseHTTPRequestHandler):
     protector = None
     backend_address = None
     timeout = None
+    
```

Comment:
why not declare these metrics in protector_main, together with the rest of them so they're all in the same place?

Reply:
actually, why not just a histogram? it is more general